what-are-bots

```diff
@@ -0,0 +1,17 @@
+
+WHY ARE THEY SO IMPORTANT?
+
+Now let me tell you a story about ChatBots and how they came to be in this ever growing internet generation, evolving gradually to this stage, gathering momentum as it approaches this point.
+
+In the beginning was Mr.Internet, the current King of transmitting information, who ruled with greatness, knowledge, and love, and basically has a lot of tricks under his sleeve. One day, he realized an organizational problem in his kingdom, and since he is a big fan of delegating and organizing, he appoints worthy Knights to handle this problem, Knighthood meaning Websites. So this was where Facebook, Google, Twitter and the rest came in.
+
+Now this improved the situation in the Kingdom, and things were awesome and his Kingdom (information) grew larger. As it grew, things became disorganized once more, and so Mr.Internet called for a meeting, discussed the problem and came up with a solution. He said Why don't you have your own specific kingdoms and territories and mansions where you can receive your guests and they all agreed to this, and then he gave them a condition saying But you must call these territories, APPS. And they were all cool with it and new knights joined their crusade (Whatsapp, SnapChat, and the rest), Now things became cooler and everyone was happy.
+
+One day, the King took a trip and when he finally got back, he noticed a new disorganization problem, this time there were so many territories, and mansions scattered all over the place, relevant and irrelevant. He was not happy with this. And so he called a meeting and stated this problem saying Why is every Knight building a mansion or territory for every single purpose when he can just have a room for it? In fact, why can't it be like a mall where you can find all you need? If a person visits Mansion A to buy cereal, why go to another Mansion B to buy Milk. Why can't Mansion B simply sell his Milk at Mansion A? You don't have to travel all around my kingdom to get what you want - he said. And that was how Bots were formed. And so Facebook, a great knight whom is popular for having many visitors offered to share visitors with any Knight who came to hang out in their mansion (messenger platform). And by visitors, I mean about 1 billion users a day.
+
+THIS IS WHY CHATBOTS ARE IDEAL FOR YOUR IDEA AND BUSINESS.
+
+So let's be honest, compare how many apps are in the app stores to how many people actually use, and why is that? It's simply because no one wants to have to DOWNLOAD and INSTALL. Messenger ChatBots solve this problem, where users get to interact with your bots without having to download or install anything. They are the new apps with a currently emerging market, where big tech companies are currently competing for a share of this market. So even as a developer or entrepreneur, you need to realize that Bots have huge potentials.
```

Comment:
I see the idea behind this last paragraph which is to make a case for why readers should embrace bot technologies. Do you think it would be better if you expounded more on these reasons?

Also reading from the top down, it felt a bit disconnected from the story. If the story has a summary or recap of some sort, it could lead fluidly into this section. Something like saying: (after the story)

 To recap:
* Websites came on board the internet to provide access location to different information
* With the boom of mobile devices, mobile apps grew to provide easy access to various experiences to users enhancing the overall internet connectivity.
* To help solve the problems of overwhelming mobile apps and to boost business (by easy customer acquisition) some big companies opened up their platform (e.g messenger, telegram, slack) to allow businesses build custom/automated experiences on them.

> You can then follow up with

Leveraging this bot platform is a great to avoid the growing [mobile app fatigue](https://techcrunch.com/2016/02/03/app-fatigue/). Of the many apps in the app stores, only a [few have user actively using them](http://fortune.com/2016/08/16/app-fatigue-is-taking-a-toll-on-smartphone-owners/). Why? It's simply because no one wants to have to DOWNLOAD and INSTALL. Chatbots solve this ... 

------

This is just an example. Feel free to modify the approach.